dali/operators/reader/loader/video/frames_decoder.h

```diff
@@ -219,6 +219,9 @@ class DLL_PUBLIC FramesDecoder {
 
   std::optional<bool> zero_latency_ = {};
 
+  // False when the file doesn't have any correct content or doesn't have valid video stream
+  bool is_valid_ = false;
```

Comment:
Nitpick1: move one field up for better packing.
Nitpick2:
```suggestion
  // False when the file doesn't have any correct content or doesn't have a valid video stream
  bool is_valid_ = false;
```

Reply:
Fixed